serveradmin/common/utils.py

```diff
@@ -28,3 +31,18 @@ def random_alnum_string(length):
         random_str = b64encode(random_bytes, b'??').decode().replace('?', '')
         if len(random_str) >= length:
             return random_str[:length]
+
+
+def profile(func):
+    @functools.wraps(func)
+    def wrapper_profile(*args, **kwargs):
+        start = time.perf_counter()
+        value = func(*args, **kwargs)
+        end = time.perf_counter()
+
+        duration = end - start
+        msg = f'{func.__name__} processed in {duration} seconds'
+        logging.getLogger('profile').info(msg)
```

Comment:
I guess it would be interesting to make the log level customisable. The style of parametrized decorators is unfortunately not that clear. If you think it would already be cool to have it, then you might want to investigate further for parametrized decorator .. or later.